docs/pageobjects_specification_files.rst

```diff
@@ -0,0 +1,76 @@
+.. _pageobjects_specification_files:
+
+PageObject Definition Files
+===========================
+
+PageObjects Specification Files use a Specific Domain Language implemented by Toolium that allows to define PageObjects
+following a human-readable specification in YAML. This specification is built and, as result of this, PageObject with
+all the expected PageElements and its configuration are loaded by Toolium, to use it as part of all test cases or
+whatever implementation using this framework.
+
+This way of test developing makes possible to "model" web pages and mobile views easily avoiding the implementation
+of custom Python classes.
+
+Toolium configuration
+---------------------
+
+To enable and configure this feature, it is needed to properly set the following configuration as part of the
+:code:`conf/properties.cfg`::
+
+
+    [PageObjectSpecFiles]
+    enabled: true
+    spec_files_path: resources/pageobject_spec_files
+    custom_page_object_module_path: common/pageobjects
+    custom_page_elements_module_path: common/pageelements
+
+* :code:`enabled`: To enable the loading of PageObjects from Specification Files.
+* :code:`spec_files_path`: Path where PageObject Specification Files are located.
+* :code:`custom_page_object_module_path`: Path (directory) where custom PageObjects are implemented in your project
+* :code:`custom_page_elements_module_path`: Path (directory) where custom PageElements are implemented in your project
```

Comment:
The last three variables are optional because they have a default value in code, could you specify it in doc?